use-timescale/data-tiering/tier-data.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Tier data to object storage
+excerpt: How to tier data to object storage
+products: [cloud]
+keywords: [data tiering]
+tags: [storage, data management]
+---
+
+import ExperimentalPrivateBeta from 'versionContent/_partials/_early_access.mdx';
+import TieringBeta from 'versionContent/_partials/_cloud-data-tiering-beta.mdx';
+
+# Tier data to object storage
+
+Tier data from primary storage to object storage to save on storage costs. You
+can continue to query a hypertable as normal after migration. All queries,
+including `JOIN`s, work as usual.
+
+This section describes how to tier data in the Timescale console. For
+information about manually tiering and untiering your data, see the
+[self-hosted data tiering][self-hosted-data-tiering] section
+
+## Turn on data tiering
+
+You can turn on data tiering from the Services Overview page in the Timescale console.
+
+<Procedure>
+
+### Turning on data tiering
+
+1.  In the Timescale console, from the `Services` list, click the name of
+    the service you want to modify.
+1.  In the `Overview` tab, locate the `Data tiering` card, and click
+    `Enable data tiering`. Confirm the action.
+1.  Data tiering can take a few seconds to start. When it has started, the
```

Comment:
Not sure whether you are saying this is how someone checks if  data tiering has been enabled.
This will show 0 after enabling as no data has been tiered yet.
Could you add the images so that I can better follow the thread here?

(Users need to take additional steps to enable this).

Reply:
updated to make it clearer.